resources/monitoring/templates/grafana/dashboards-1.14/apiserver.yaml.bak

```diff
@@ -0,0 +1,1734 @@
+{{- /*
+Generated from 'apiserver' from https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/master/manifests/grafana-dashboardDefinitions.yaml
```

Comment:
Why does this file have `.bak` suffix?

Reply:
All dashboards have been replaced with the ones from the upstream chart, except `apiserver` and `kubelet`. Those two depend on new metrics coming from the new versions of the exporters. Since we decided to handle subschart upgrades separately, we cannot use the new `apiserver` and `kubelet` dashboards and have to use the old ones. I decided to keep the new ones with a `bak` suffix (so they won't be rendered) and switch to them once we upgrade the exporters.